Modules/_posixsubprocess.c

```diff
@@ -98,15 +98,17 @@ _enable_gc(int need_to_reenable_gc, PyObject *gc_module)
         PyErr_Fetch(&exctype, &val, &tb);
         result = PyObject_CallMethodNoArgs(
             gc_module, _posixsubprocessstate_global->enable);
+        if (result == NULL) {
+            /* We might have created a child process at this point, we
+             * we have no good way to handle a failure to reenable GC
+             * and return information about the child process. */
+            PyErr_Print();
```

Comment:
I changed this from PyErr_Clear to PyErr_Print so that the user would at least have some way to know that something serious an unusual went wrong.